config.default.yaml

```diff
@@ -196,18 +216,19 @@ transformers:
 atlite:
   nprocesses: 4
   cutouts:
-    # geographical bounds automatically determined from countries input
+    # use 'base' to determine geographical bounds and time span from config
+    # base:
+    # module: era5
```

Comment:
why is this line added here?

Reply:
I can deleted it again, it should just show the flexible option to create a cutout from the network itself instead of setting the coordinates manually